### ISSUE TITLE
Padding formatting is not documented?

### ISSUE THREAD

**Issue by slhck** [Delgan/loguru#637]
Great library, thanks for making this!

I found [this guide](https://loguru.readthedocs.io/en/stable/resources/recipes.html#dynamically-formatting-messages-to-properly-align-values-with-padding) where some examples for padding strings are given:

```python
fmt = "{time} | {level: <8} | {name: ^15} | {function: ^15} | {line: >3} | {message}"
```

However, I don't know what `<`, `^` and `>` mean here. It is not explained in the section.

I also cannot find any mention of these symbols in [the API docs](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.add).

Would it be possible to add a short explainer there?

**Comment by slhck** [comment]
Oh, I get it now … are these left, center, and right alignment, respectively?

**Comment by Delgan** [comment]
This is using the Python built-in formatting rules (same as the [`str.format()`](https://docs.python.org/3/library/stdtypes.html#str.format) method), see this guide for example: [Using `%` and `.format()` for great good!](https://pyformat.info/).

**Comment by slhck** [comment]
Thanks. I made a PR to make the documentation a bit clearer with respect to that (I searched the API page for "align" or "pad" but was unable to find anything).

**Comment by Delgan** [comment]
Fixed by #638.
